internal/serverinstall/ecs.go

```diff
@@ -158,7 +158,14 @@ func (i *ECSInstaller) Install(
 				return err
 			}
 
-			if netInfo, err = awsinstallutil.SetupNetworking(ctx, ui, sess, i.config.Subnets); err != nil {
+			grpcPort, _ := strconv.Atoi(serverconfig.DefaultGRPCPort)
+			httpPort, _ := strconv.Atoi(serverconfig.DefaultHTTPPort)
+			ports := []*int64{
+				aws.Int64(int64(grpcPort)),
+				aws.Int64(int64(httpPort)), // TODO: Not needed for runner install
```

Comment:
What is `TODO` about this? Just curious. The Runner install doesn't doesn't seem to include it or the `grpcPort`

Reply:
The ports set here ultimately result in a security group being created that is shared by both the server and the runner. The server needs the HTTP port but the runner has no use for it. The TODO is getting at that we might want to split them up somehow, just to tighten the SG a bit.

Reply:
👍